docs/pages/installation/kubernetes.md

```diff
@@ -58,6 +58,21 @@ Run the following:
  </div></div>
  </pre>
 
+Customize of deployment the Meshery adapters:
+
+ <pre class="codeblock-pre">
+ <div class="codeblock"><div class="clipboardjs">
+ $ helm repo add meshery https://meshery.io/charts/
+
+ # Manually disable the adapters you do not want to deploy
+ $ helm install --set key=value --set key=value my-meshery meshery/meshery --version 2.1.2
```

Comment:
Thanks for this, @Aisuko. For as much as we can, we encourage Meshery deployments to subscribe to either the `stable-latest` or `edge-latest` release channels. Is it possible to have `stable-latest` as the default version in the Helm chart?

Reply:
Did you mean the Docker image of Meshery, I believe we have https://github.com/layer5io/meshery/blob/ecceb1a296c48205c3b8759bebe27fd7d58f1d11/install/kubernetes/helm/meshery/values.yaml#L9

And you mean that we can support `helm install` support set `statble` and `edge` ?

Reply:
@Aisuko, thanks for clarifying. Having reflected on this a little bit more (in today's Meshery Development meeting), the suggestion coming out of the meeting was to create two independent versions of the chart, resulting in two entries for Meshery on Artifact Hub: stable and edge (see Linkerd project as an example).

The other consensus formed was that of consolidating the Helm chart version to be the same as that of Meshery version that releases it, so that there's no confusion for users between what Meshery version they are running and what Helm chart version they are running: both would be the same. Can we move to this going forward?

Do both of these make sense / sound good?

Reply:
Thank you told me this, sir. That sounds good. I believe that makes sense. I will do PR for this, but this PR only includes en/disable meshery adapters,we can merge at first.

Reply:
Ok, sounds real good. 👍